packages/vats/src/lib-chainStorage.js

```diff
@@ -100,16 +96,13 @@ const makeNullStorageNode = () => {
  * falling back to an inert object with the correct interface (but incomplete
  * behavior) when that is unavailable.
  *
- * @param {ERef<ChainStorageNode?>} chainStorage
- * @param {string} [childName]
+ * @param {ERef<StorageNode?>} chainStorage
+ * @param {string} childName
  * @returns {Promise<StorageNode>}
  */
-export async function makeStorageNode(chainStorage, childName) {
+export async function makeStorageNodeChild(chainStorage, childName) {
```

Comment:
I support this rename, and would recommend taking it farther:
```suggestion
export async function makeStorageNodeChild(storageNode, childName) {
```